### PR TITLE
profiles: fix misc in kmail and transmission-qt & add kontact.profile

### PR DESCRIPTION
**Pull request by Sadoon-AlBader**
This removes "nosound" from kmail.profile to allow notification sounds, and also adds a "dbus-user.talk org.freedesktop.Notifications" to allow notifications to start.
Also adds kontact.profile based on kmail.profile, have been using it and works well.
Also edits transmission-qt.profile to allow systray, commented out as is usually the case with profiles, but better to at least let the user know that it might be necessary to add to .local if they need systray.

EDIT: if the maintainers also prefer to keep "nosound" for kmail, it might make sense to also add it to kontact.profile since I didn't add it there to start, I think it makes sense to either remove it or add a comment letting the user know that sound notifications won't work unless they "ignore nosound"

### CODE REVIEW

File: etc/profile-a-l/kontact.profile [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Firejail profile for kontact
```
Comment (rusty-snake) [inline etc/profile-a-l/kontact.profile:1]: Firecfg?

File: etc/profile-m-z/transmission-qt.profile [hunk 2]
```diff
@@ -15,6 +15,9 @@ private-bin transmission-qt
 #dbus-user.own com.transmissionbt.Transmission.*
 #dbus-user.talk org.freedesktop.Notifications
 
+# If you need system tray support, add this line to your transmission-qt.local
```
Comment (rusty-snake) [inline etc/profile-m-z/transmission-qt.profile:18]: I would just enabled it with the condition.

File: etc/profile-a-l/kmail.profile [hunk 3]
```diff
@@ -63,4 +62,7 @@ private-dev
 # writable-run-user is needed for signing and encrypting emails
 writable-run-user
 
+# enable desktop notifications
+dbus-user.talk org.freedesktop.Notifications
```
[the remark this answers is not in the document]
Comment (Sadoon-AlBader) [reply etc/profile-a-l/kmail.profile:66]: In this case should the user be prompted to add both that line and `dbus-user filter` to their .local?

File: etc/profile-a-l/kontact.profile (hunk 1 above)
Comment (Sadoon-AlBader) [reply etc/profile-a-l/kontact.profile:1]: I'll add that, thanks for reminding me, added to my system but forgot to do so here.

File: etc/profile-m-z/transmission-qt.profile (hunk 2 above)
Comment (Sadoon-AlBader) [reply etc/profile-m-z/transmission-qt.profile:18]: Will fix, thanks.

**Comment by Sadoon-AlBader** [comment]
@rusty-snake All your suggestions are now implemented, thanks again :)

**Review by kmk3 — CHANGES_REQUESTED** [review]
The new profile is nearly identical to kmail.profile; it should probably be a
redirect to the latter.

See redirect_alias-profile.template.

File: etc/profile-a-l/kmail.profile [hunk 4]
```diff
@@ -63,4 +62,8 @@ private-dev
 # writable-run-user is needed for signing and encrypting emails
 writable-run-user
 
+# to enable desktop notifications, add these commented lines to your kmail.local file
+#dbus-user.talk org.freedesktop.Notifications
+#dbus-user filter
```
Comment (rusty-snake) [inline etc/profile-a-l/kmail.profile:67]:
```suggestion
#dbus-user filter
#dbus-user.talk org.freedesktop.Notifications
```

File: etc/profile-a-l/kmail.profile [hunk 5]
```diff
@@ -63,4 +62,8 @@ private-dev
 # writable-run-user is needed for signing and encrypting emails
 writable-run-user
 
+# to enable desktop notifications, add these commented lines to your kmail.local file
```
Comment (rusty-snake) [inline etc/profile-a-l/kmail.profile:65]:
```suggestion
# To harden kmail.profile, add the following lines to your kmail.local file
```

**Review by rusty-snake — COMMENTED** [review]
Now I had time for the full review.

**Pull request merged**